stable/admin/values.yaml

```diff
@@ -49,7 +49,7 @@ nuodb:
   image:
     registry: docker.io
     repository: nuodb/nuodb-ce
-    tag: 4.2.1
+    tag: '4.2'
```

Comment:
The pull policy is `IfNotPresent` so I don't think it would cause a rolling upgrade. But it may cause a database to be started in a version coexistence mode where the existing processes are running with the previous `4.2` and new processes are started with the latest `4.2`. We would want to avoid that, so I will change this to specify 4.2.4.vee.